lang/rust/avro/src/schema.rs

```diff
@@ -611,8 +611,8 @@ impl RecordField {
                 aliases
                     .iter()
                     .flat_map(|alias| alias.as_str())
-                    .map(|alias| Alias::new(alias).expect("Alias is not valid"))
-                    .collect::<Vec<Alias>>()
+                    .map(|alias| alias.to_string())
+                    .collect::<Vec<String>>()
```

Comment:
Where did you find the information about Aliases not requiring namespaces for `RecordField`? I was using the same `alias parser` to parse an `Enum` for example, but they now have different signature, it's not really a problem, but in the spec the text is the same for `Enum` and for `RecordField`

Reply:
Named schema's `aliases` are for the schema's name which might be namespaced.
Record field's aliases are for the field's `name` which is not namespaced. The field's type might be a (namespaced) reference to Schema.